tests/testcommon.py

```diff
@@ -229,9 +229,15 @@ class SampleParamsHolder(ParamsBase):
     handling of meta parameters like `frompackages`, `packages`, `params`, `lines`
     in inherited classes
     """
+
+    packages = (
+        ('datetime', 'dt'),
```

Comment:
`datetime` is pretty often used package and could be easily imported at the module level inside the testcommon.py in the future for some unrelated purpose.  Suggest using more rarely used package for this matter (like 'fractions' or 'bisect' or something similar)

Reply:
Done. Thanks!